.pre-commit-config.yaml

```diff
@@ -30,11 +30,11 @@ repos:
         files: ^pandas
         exclude: ^pandas/tests
         args: [--select, "ANN001,ANN2", --fix-only, --exit-non-zero-on-fix]
+    -   id: ruff
```

Comment:
If we need a new section for pylint for `--preview`, could you rename this `ruff-pylint`?

Reply:
The id is defined in `ruff-pre-commit`, not a name we can change, changing this will lead to the error below.

```
[ERROR] `ruff-pylint` is not present in repository https://github.com/astral-sh/ruff-pre-commit.  Typo? Perhaps it is introduced in a newer version?  Often `pre-commit autoupdate` fixes this.
```